scaleway/resource_lb_frontend_beta.go

```diff
@@ -164,3 +344,16 @@ func resourceScalewayLbFrontendBetaDelete(d *schema.ResourceData, m interface{})
 
 	return nil
 }
+
+func aclEquals(aclA, aclB *lb.ACL, skipNameVerification bool) bool {
+	if !skipNameVerification && aclA.Name != aclB.Name {
```

Comment:
Why do we need this special case `skipNameVerification` ?

Reply:
It was mostly to deal with the case where the requested name is empty (and we generate a new one). It's possible that we can drop it, but let me have working integration tests first, and I will recheck it again. (once the issue below is fixed).

Reply:
Tests are working, and I've been able to drop this override.